x/auth/client/testutil/suite.go

```diff
@@ -1548,317 +1548,317 @@ func (s *IntegrationTestSuite) TestSignWithMultiSignersAminoJSON() {
-				fmt.Sprintf("--%s=%s", flags.FlagChainID, "foobar"),
-			},
-			expectErrBroadCast: true,
-		},
-		{
-			name:     "wrong denom in tip: error",
-			tipper:   tipper,
-			feePayer: feePayer,
-			tip:      sdk.Coin{Denom: fmt.Sprintf("%stoken", val.Moniker), Amount: sdk.NewInt(0)},
-			tipperArgs: []string{
-				fmt.Sprintf("--%s=%s", flags.FlagTip, "1000wrongDenom"),
-				fmt.Sprintf("--%s=%s", flags.FlagSignMode, flags.SignModeDirectAux),
-				fmt.Sprintf("--%s=true", flags.FlagAux),
-			},
-			feePayerArgs: []string{
-				fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
-				fmt.Sprintf("--%s=%s", flags.FlagSignMode, flags.SignModeDirect),
-				fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
-				fmt.Sprintf("--%s=%s", flags.FlagFrom, feePayer),
-				fmt.Sprintf("--%s=%s", flags.FlagFees, fee.String()),
-			},
-			errMsg: "insufficient funds",
-		},
-		{
-			name:     "insufficient fees: error",
-			tipper:   tipper,
-			feePayer: feePayer,
-			tip:      sdk.Coin{Denom: fmt.Sprintf("%stoken", val.Moniker), Amount: sdk.NewInt(0)},
-			tipperArgs: []string{
-				fmt.Sprintf("--%s=%s", flags.FlagTip, tip),
-				fmt.Sprintf("--%s=%s", flags.FlagSignMode, flags.SignModeDirectAux),
-				fmt.Sprintf("--%s=true", flags.FlagAux),
-			},
-			feePayerArgs: []string{
-				fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
-				fmt.Sprintf("--%s=%s", flags.FlagSignMode, flags.SignModeDirect),
-				fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastSync),
-				fmt.Sprintf("--%s=%s", flags.FlagFrom, feePayer),
-			},
-			errMsg: "insufficient fees",
-		},
-	}
-
-	for _, tc := range testCases {
-		tc := tc
-		s.Run(tc.name, func() {
-			res, err := govtestutil.MsgSubmitLegacyProposal(
-				val.ClientCtx,
-				tipper.String(),
-				"test",
-				"test desc",
-				govtypes.ProposalTypeText,
-				tc.tipperArgs...,
-			)
-
-			if tc.expectErrAux {
-				require.Error(err)
-			} else {
-				require.NoError(err)
-				genTxFile := testutil.WriteToNewTempFile(s.T(), string(res.Bytes()))
-
-				// broadcast the tx
-				res, err = TxAuxToFeeExec(
-					val.ClientCtx,
-					genTxFile.Name(),
-					tc.feePayerArgs...,
-				)
-				switch {
-				case tc.expectErrBroadCast:
-					require.Error(err)
-
-				case tc.errMsg != "":
-					require.NoError(err)
-
-					var txRes sdk.TxResponse
-					require.NoError(val.ClientCtx.Codec.UnmarshalJSON(res.Bytes(), &txRes))
-
-					require.Contains(txRes.RawLog, tc.errMsg)
-
-				default:
-					require.NoError(err)
-
-					var txRes sdk.TxResponse
-					require.NoError(val.ClientCtx.Codec.UnmarshalJSON(res.Bytes(), &txRes))
-
-					require.Equal(uint32(0), txRes.Code)
-					require.NotNil(int64(0), txRes.Height)
-
-					bal = s.getBalances(val.ClientCtx, tipper, tc.tip.Denom)
-					tipperInitialBal = tipperInitialBal.Sub(tc.tip)
-					require.True(bal.Equal(tipperInitialBal.Amount))
-				}
-			}
-		})
-	}
-}
+// func (s *IntegrationTestSuite) TestAuxSigner() {
```

Comment:
I think we should uncomment everything and add something like `s.T().Skip() // TODO to re-enable in #12274` instead in the first line of the test.